src/applicationContext.ts

```diff
@@ -217,9 +223,14 @@ export class ApplicationContext {
 		this.dateRangeCommandValidator = new Validator<DateRangeCommand>(this.dateRangeCommandFactory)
 		this.dateRangeFactory = new DateRangeFactory()
 		this.dateRangeValidator = new Validator<DateRange>(this.dateRangeFactory)
+		this.eventRecordFactory = new EventRecordFactory()
+
+		this.learnerRecordConfig = new LearnerRecordConfig('http://localhost:9000')
```

Comment:
hardcoded var?

Reply:
@m24murray back end issue, no idea where this came from maybe @Peter-D-Taylor or @jakub-kainos?